src/predict.Rmd

```diff
@@ -133,6 +133,32 @@ As mentioned above, the predicted posterior probabilities can be accessed via th
 head(getPredictionProbabilities(pred))
 ```
 
+### Extract Standard Errors
+
+Some Learners provide standard errors for predictions, which can be accessed in mlr. An overview is given by calling the function[&listLearners]. In order to include learners from packages which are not installed in the overview, one has to set ``check.packages``  as ``FALSE``.
+
+```{r, results='hide', warning=FALSE}
+listLearners("regr", check.packages = FALSE)
+```
+
+In this example we train a Gaussian Process Regression on the Boston Housing dataset. Since the Gaussian Process, ``regr.km``, does not allow factor data, we have to use only numerical variables  for the training. For the learner, the ``predict.type`` has to be specified as correctly:
```

Comment:
We used GPs to show that the package allows to use this powerful method, that to my knowledge is often used because it gives standard errors and confidence intervals. A simpler model might be more suitable however, it has been corrected in the new commit.